unittests/web/RPCServerHandlerTests.cpp

```diff
@@ -129,7 +129,8 @@ TEST_F(WebRPCServerHandlerTest, WsNormalPath)
     session->upgraded = true;
     static auto constexpr request = R"({
                                         "command": "server_info",
-                                        "id": 99
+                                        "id": 99,
+                                        "api_version": 2
```

Comment:
Don't we need to add a test with `api_version: 1`?

Reply:
api_version may not matter. It should be added to response once "api_version" presents in request.